src/openrct2/interface/viewport.c

```diff
@@ -1032,26 +1029,21 @@ static void store_interaction_info(paint_struct *ps)
 /**
  * rct2: 0x00679236, 0x00679662, 0x00679B0D, 0x00679FF1
  */
-static bool sub_679236_679662_679B0D_679FF1(uint32 ebx, rct_g1_element *image, uint8 *esi) {
+static bool sub_679236_679662_679B0D_679FF1(uint32 imageType, rct_g1_element *image, uint8 *index) {
```

Comment:
This should be renamed to make it clear that this is testing to see if a pixel can be interacted with.

Reply:
I wasn't exactly sure what its doing, any suggestions on the name?